app/plugin_commands.go

```diff
@@ -136,6 +137,12 @@ func (a *App) tryExecutePluginCommand(args *model.CommandArgs) (*model.Command,
 		return nil, nil, nil
 	}
 
+	// Checking if plugin is working or not
+	if err := pluginsEnvironment.PerformHealthCheck(matched.PluginId); err != nil {
+		message := fmt.Sprintf("Plugin for the /%s command is not working. Please contact your system administrator.", trigger)
+		return matched.Command, nil, model.NewAppError("ExecutePluginCommand", message, nil, "err= Plugin failed to start: "+matched.PluginId, http.StatusInternalServerError)
```

Comment:
`Plugin failed to start` doesn't seem like the right error here. I believe this scenario is when:

- Plugin is enabled
- Plugin recently crashed and the plugin health check job has not picked up on that yet

Maybe something like `Plugin has recently crashed`?